lesson_07/homeworks_07.py

```diff
@@ -1,25 +1,27 @@
+import random
 # task 1
 """ Задача - надрукувати табличку множення на задане число, але
 лише до максимального значення для добутку - 25.
 Код майже готовий, треба знайти помилки та випраавити\доповнити.
 """
-def multiplication_table(number):
+def multiplication_table(number: int):
     # Initialize the appropriate variable
     multiplier = 1
 
-    # Complete the while loop condition.
-    while multiplier <= number:
+    # Complete the while loop condition. Loop starts with positive number only.
+    while number > 0:
         result = number * multiplier
-        # десь тут помила, а може не одна
-        if  result > "25":
+        # десь тут помилка, а може не одна
```

Comment:
```suggestion
        # десь тут помилка, а може не одна
```
знайти помилку в слові помилка - безцінно, для іного є віза )))